src/DependencyInjection/SonataAdminExtension.php

```diff
@@ -97,6 +97,11 @@ public function load(array $configs, ContainerBuilder $container)
         $pool->replaceArgument(1, $config['title']);
         $pool->replaceArgument(2, $config['title_logo']);
         $pool->replaceArgument(3, $config['options']);
```

Comment:
You're replacing argument 1, 2 and 3 in both `AddDependencyCallsCompilerPass.php ` and here but with different values. 

Shouldn't these lines be removed in favor of the `setDeprecatedPropertiesForBC` call ?

Reply:
I left them intentionally just in case some weird BC break, just to be safe, but I guess they can be removed.